node/src/chain_specs/manta.rs

```diff
@@ -157,30 +167,43 @@ fn manta_dev_genesis(
         sudo: manta_runtime::SudoConfig {
             key: Some(root_key),
         },
+        parachain_staking: ParachainStakingConfig {
+            candidates: invulnerables
+                .iter()
+                .cloned()
+                .map(|(account, _)| {
+                    (
+                        account,
+                        manta_runtime::staking::NORMAL_COLLATOR_MINIMUM_STAKE,
+                    )
+                })
+                .collect(),
+            delegations,
+            inflation_config: manta_runtime::staking::inflation_config::<manta_runtime::Runtime>(),
+        },
         parachain_info: manta_runtime::ParachainInfoConfig {
             parachain_id: MANTA_PARACHAIN_ID.into(),
         },
         collator_selection: manta_runtime::CollatorSelectionConfig {
             invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
-            candidacy_bond: MANTA * 10000, // How many tokens will be reserved as collator
+            candidacy_bond: 10_000 * MANTA, // How many tokens will be reserved as collator
             ..Default::default()
         },
         session: manta_runtime::SessionConfig {
             keys: invulnerables
                 .iter()
                 .cloned()
-                .map(|(acc, aura)| {
+                .map(|(acc, session_keys)| {
                     (
-                        acc.clone(),              // account id
-                        acc,                      // validator id
-                        manta_session_keys(aura), // session keys
+                        acc.clone(),  // account id
+                        acc,          // validator id
+                        session_keys, // collator session keys
                     )
                 })
                 .collect(),
         },
-        aura_ext: Default::default(),
         parachain_system: Default::default(),
-        polkadot_xcm: manta_runtime::PolkadotXcmConfig {
+        polkadot_xcm: calamari_runtime::PolkadotXcmConfig {
```

Comment:
use manta_runtime?

Reply:
thanks, this was a copy-pasta mistake. I fixed it in #952